ui/ducks/metamask/metamask.js

```diff
@@ -49,6 +49,7 @@ const initialState = {
     showTestNetworks: false,
     useNativeCurrencyAsPrimaryCurrency: true,
     petnamesEnabled: true,
+    featureNotificationsEnabled: false,
```

Comment:
I've copied what the experimental petnames changed.

I've not been onboarded to the project, mostly winging it 🤠 

What does this `ducks` do? Is this just for the initial redux state until it uses the state inside the preferences controller?

Reply:
There is some stuff in here that is actually used and requires more scrutiny but my blanket statement for what this duck does is "confuse our contributors" :P It was at one point used as a sort of "optimistic update" aka if we wanted to change something in the background we would also change it here optimistically to update the UI early instead of waiting for the return trip from the state update from the background.